components/TheDataExperience.vue

```diff
@@ -205,22 +249,30 @@ export default {
     }
   },
   watch: {
-    fileManager() {
-      if (this.fileManager === null) {
-        this.tab = 'load-data'
-        this.scrollToTop()
+    fileManager(value) {
+      if (value === null) {
+        this.switchTab('load-data')
         this.success = false
         this.progress = false
         this.error = false
       }
+    },
+    // debounce overlay
+    experienceProgress: {
+      immediate: true,
+      handler: debounce(function (value) {
+        this.overlay = value
+      }, 200)
     }
   },
   mounted() {
-    this.$root.$on('goToFileExplorer', () => {
-      this.tab = 'file-explorer'
-    })
+    this.switchTab('load-data')
```

Comment:
This is to ensure that the first tab is always shown when the component is mounted...